tests/main.test.ts

```diff
@@ -16,26 +16,26 @@ describe("Devpool Updates", () => {
   beforeAll(async () => {
     // @ts-expect-error main() is only exported for testing
     main = await (await import("../index")).main;
-    // silence stderr since we expect errors to be logged
-    jest.spyOn(console, "warn").mockImplementation(jest.fn());
   });
 
   beforeEach(() => {
     drop(db);
 
     // Add all the default repos of the organization
-    for (let i = 0; i < opt.out.length; ++i) {
-      const repoName = opt.out[i].split("/")[1];
-      db.repo.create({
-        id: i + 1,
-        owner: DEVPOOL_OWNER_NAME,
-        name: repoName,
-        html_url: `https://github.com/${DEVPOOL_OWNER_NAME}/${repoName}`,
-      });
+    const repos = opt.in.concat(opt.out);
```

Comment:
Why are you combining opt out if those are explicitly meant to be excluded

Reply:
https://github.com/ubiquity/devpool-directory/commit/025868c32c5571f6c3457f46d1adce3b1d7aa8cf

It comes from this commit, I must have made a slight mistake in how I've merged it that should be eresdev's commit

Reply:
@EresDev rfc

Reply:
> @EresDev rfc

This part had multiple problems and I tried to address some with this fix I didn't want to put more time into this because the work I was doing was unrelated to this. But let's see what is needed now. 

- before you use in, and exclude out, this part needs to deal with out/in with a single value. It was treating properly the fully qualified repository name "GithubAccountName/RepostioryName", but if it was just "GithubAccountName", it treated it as "RepostioryName" That is why I had to put a check to only use a fully qualified repository name. This needs to be fixed. https://github.com/ubiquity/devpool-directory/blob/131e16d78d245c1f4bf03e31b6935ad78900d450/tests/main.test.ts#L28
- after the above fix, use the in, and exclude the out. 
- also move the mocks folder out of the root directory into the test folder. mocks are only used by tests so they should live together.

Reply:
One suggestion is always to use a full repository name in opt.json "GithubAccountName/RepostioryName", and it will make out/in redundant. This will result in lesser & cleaner code, less requests, and clear intent. But it is just a suggestion as this will require redoing a few parts and I was not the part of initial discussion on this. But @pavlovcik has the final say on this.

Reply:
No it definitely needs to be able to automatically include new repositories for privileged organizations like @ubiquity. 

This means that we must support org only syntax.

Reply:
I have not looked in this PR for quite some time, but receiving a dozen notification emails a day about closing and reopening tasks I feel I will join the reviewers party.

About opt in and opt out, I initially implemented quite a simple solution at https://github.com/ubiquity/devpool-directory/pull/982/files that worked correctly at least to extent to the tests coverage at that time. Perhaps this can be refactored but it did seem to work, first by adding repositories, then deleting opt outs from the set. I did not follow up the recent changes, what was the main issue with the previous approach?

Reply:
> > I have not looked in this PR for quite some time, but receiving a dozen notification emails a day about closing and reopening tasks I feel I will join the reviewers party.
> 
> Unless you're very partial towards using email, I highly recommend you use github.com/notifications ! This is because you only need to "pull" (instead of push) the latest notification in each thread. Emails give you multiple "push" notifications for every activity in every thread. So you have a significant higher amount of clutter to sift through.
> 

The email was good enough till recently. Great tip with the notifications, I will check this out asap!